examples/asa/scripts/transfer/contract-owned-asa/change-contract-owner.js

```diff
@@ -3,8 +3,8 @@
  * This file demonstrates how to change owner of ASA owned by
  * smart contract account(stateless).
  */
-const { executeTransaction, stringToBytes, addressToPk } = require('@algo-builder/algob');
-const { types } = require('@algo-builder/runtime');
+const { executeTransaction } = require('@algo-builder/algob');
+const { types, stringToBytes, addressToPk } = require('@algo-builder/runtime');
```

Comment:
I would prefer that user only imports `algob` in scripts  (could be exported as `algob.convert`). 
`runtime` is more for testing or internal code.

Reply:
OK.